packages/ui-components/src/components/HeaderContainer/HeaderContainer.stories.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+
+import { HeaderContainer, HeaderContainerProps } from "./index"
+
+export default {
+  title: "Internal/HeaderContainer",
+  component: HeaderContainer,
+  argTypes: {
+    children: {
```

Comment:
**(Blocking Improvement)**

Storybook doesn't pick up type `ReactNode` for children and reverts to `any`. Here's an [improvement](https://github.com/cloudoperators/juno/pull/601/commits/f1fa2903825034370c3449a14afaf62ae6f69686).